src/Sulu/Bundle/SecurityBundle/Admin/SuluSecurityContactsContentNavigationProvider.php

```diff
@@ -2,30 +2,36 @@
 
 namespace Sulu\Bundle\SecurityBundle\Admin;
 
-use Sulu\Bundle\AdminBundle\Navigation\ContentNavigationInterface;
+use Sulu\Bundle\AdminBundle\Navigation\ContentNavigationProviderInterface;
 use Sulu\Bundle\AdminBundle\Navigation\ContentNavigationItem;
 use Sulu\Component\Security\Authorization\SecurityCheckerInterface;
 
-class SuluSecurityContactsContentNavigation implements ContentNavigationInterface
+class SuluSecurityContactsContentNavigationProvider implements ContentNavigationProviderInterface
```

Comment:
I would recommend shortening this suffix generally to `NavigationProvider`, context can be inferred from the namespace.